desktop/release-notes.md

```diff
@@ -33,6 +33,60 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/ge
 
 ### New
 
+- Introduce the new Dev Environments CLI `docker dev`.
+- Docker Desktop can now be installed to any drive and folder using the `--installation-dir`. Partially addresses [docker/roadmap#94](https://github.com/docker/roadmap/issues/94).
+- Implemented enhanced-container-isolation feature as part of Docker Desktop's hardening-desktop effort (public link to be provided).
```

Comment:
We sort of do...but since the docs for ECI aren't published yet it'll cause the build checks to fail. It's easier if we publish all the docs and release notes for 4.13 first and then I quickly create a new PR to add the right links post-release

Reply:
Can we change to:

Two new security features have been introduced for Docker Business users, Settings Management and Enhanced Container Isolation. Read more about Docker Desktop’s new Hardened Desktop security model [here](https://docs.docker.com/desktop/hardened-desktop/).

Reply:
Also I'd suggest moving this to the top of the bullet point list.